.github/workflows/daily.yml

```diff
@@ -377,7 +377,7 @@ jobs:
     - name: testprep
       run: |
         sudo apt-get install vmtouch
-        mkdir /tmp/master 
+        mkdir /tmp/master
```

Comment:
Maybe later sometime but at some point please cleanup unnecessary changes. It pollutes blame log and makes it harder to cherry-pick the commit for fork maintainers.

Reply:
Reverted unnecessary changes.